.github/workflows/k8s-integration-tests.yml

```diff
@@ -0,0 +1,35 @@
+name: K8S integration tests
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+
+        steps:
+        - uses: actions/checkout@v3
+          with:
+            ref: ${{github.head_ref}}
+            fetch-depth: 0
+            repository: ${{ github.event.pull_request.head.repo.full_name || github.repository }}
+        - uses: jupyterhub/action-k3s-helm@v3
+          with:
+            docker-enabled: true
+
+        - name: Docker Build
+          run: docker build -t jasminegraph .
+
+        - name: Install python K8S client
+          run: pip install kubernetes
+
+        - name: Grant default user permissions
+          run: kubectl apply -f ./.github/workflows/resources/test-rbac.yaml
```

Comment:
Lets fix this path. Based on the offline discussion we had the folder name is supposed to be `.github`

Reply:
Yes. Isn't this the correct path?.
From current path it is refering to the content inside the `.github` folder

Reply:
I see, then this is fine.